### ISSUE TITLE
Null value for OID

### ISSUE THREAD

**Issue by fkleedorfer** [researchstudio-sat/webofneeds#691]
Sometimes, we have these error messages in the js console. Not sure what specific behaviour precedes or follows this.

```
linkeddata-service-won.js: fetching:        https://satsrv06.researchstudio.at/won/resource/need/4508223599103297500/connections/
app_jspm.bundle.js:60591 error in lexicon retrieving OID:
app_jspm.bundle.js:60592 403
app_jspm.bundle.js:60601 Null value for OID
undefined:1 Uncaught (in promise) Error: Failed to fetch connection uris for need <https://satsrv06.researchstudio.at/won/resource/need/4508223599103297500> because of:
Unknown retrieving OID in lexicon:403
    at Lexicon.Lexicon.retrieve (https://satsrv06.researchstudio.at/owner/generated/app_jspm.bundle.js:60603:15)
    at QueryEngine.QueryEngine.denormalizeBindings (https://satsrv06.researchstudio.at/owner/generated/app_jspm.bundle.js:84604:36)
    at QueryEngine.QueryEngine.denormalizeBindingsList (https://satsrv06.researchstudio.at/owner/generated/app_jspm.bundle.js:84561:27)
    at https://satsrv06.researchstudio.at/owner/generated/app_jspm.bundle.js:84696:33
    at QueryEngine.QueryEngine.executeSelect (https://satsrv06.researchstudio.at/owner/generated/app_jspm.bundle.js:84797:15)
    at QueryEngine.QueryEngine.executeQuery (https://satsrv06.researchstudio.at/owner/generated/app_jspm.bundle.js:84693:14)
    at QueryEngine.QueryEngine.execute (https://satsrv06.researchstudio.at/owner/generated/app_jspm.bundle.js:84636:16)
    at Store.Store.graph (https://satsrv06.researchstudio.at/owner/generated/app_jspm.bundle.js:86781:19)
    at https://satsrv06.researchstudio.at/owner/generated/app_jspm.bundle.js:88692:55
    at https://satsrv06.researchstudio.at/owner/generated/app_jspm.bundle.js:84731:17
```

**Comment by fkleedorfer** [comment]
the exception is caused here: https://github.com/researchstudio-sat/webofneeds/blob/818fa31865417c4750d8082db54354ae4451cf3f/webofneeds/won-owner-webapp/src/main/webapp/app/service/linkeddata-service-won.js#L1557

**Comment by m0ru** [comment]
Might be fixed by the changes proposed in #693
